src/pykeen/nn/message_passing.py

```diff
@@ -431,9 +420,106 @@ def forward(
                 m = m * weights_r.unsqueeze(dim=1).unsqueeze(dim=2)
 
             # message aggregation
-            out.index_add_(dim=0, index=target_r, source=m)
+            accumulator.index_add_(dim=0, index=target_r, source=m)
+
+        return accumulator.view(-1, self.output_dim)
+
+
+class RGCNLayer(nn.Module):
+    """One RGCN layer."""
```

Comment:
can you give some more high-level dscription of this? like perhaps a math equation?

Reply:
782bb2c 

RST syntax for math and links may be wrong :sweat_smile: